doc/developer_guide/psy_data.rst

```diff
@@ -655,23 +674,28 @@ verification library (taken from ``lib/read_only/lfric/Makefile``):
         ../../process.py $(PROCESS_ARGS) $< > psy_data_base.f90
 
 This will create the processed file ``psy_data_base.f90`` in the directory
-of the library (which has the advantage that you will be using consistent
-compiler settings in your library, and consistent parameter specifying
-the required types and dimensions). You still need to declare all these
-automatic functions in your generic interface for the various functions.
-It is recommended to use a jinja template as well.
+of the library, where it will be compiled. Having a separate pre-processed
+source code version of the base class for each library (as opposed to one
+compiled base class library that is used for all libraries) has the advantage
+that consistent compiler settings will be used in your library, and consistent
+parameters have been provided specifying the required types and dimensions.
 
 The ``process.py`` script provides the two variables for a template
-(based on its parameters of course): ``ALL_DIMS`` stores the list of 
+(based on its command line parameters of course): ``ALL_DIMS`` stores the list of 
 required dimensions, and ``ALL_TYPES`` stores the type information
 requested when invoking ``process.py``. ``ALL_TYPES`` is a three-tuple
-that includes the name to use, the Fortran data type, and number of
-bits for the data type. While number of bits is not used in the base class,
-the read-only-verification base class (see
+that lists the name to use when creating subroutine names, the Fortran data
+type, and number of bits for the data type. While number of bits is not used
+in the base class, the read-only-verification base class (see
 :ref:`psydata_read_only_base_class`) uses it. If more types are required,
 they can be defined in ``process.py``. If the additional types need
-different number of bits and are required in a read-only library, the
-read-only-verification base class needs to be adjusted as well.
+different numbers of bits and are required in a read-only library, the
+read-only-verification base class (``lib/read_only/read_only_base.jinja``)
+needs to be adjusted as well. 
+
+Below is a short excerpt that shows how these variables are defined by
+default, and how they are used to create subroutines and declare their
+parameters in ``lib//read_only_base.jinja``:
```

Comment:
Did you intend the double slash '//' here?

Reply:
No :)